react/features/breakout-rooms/actions.js

```diff
@@ -170,6 +171,11 @@ export function moveToRoom(roomId?: string) {
             _roomId.domain = domainParts.join('@');
         }
 
+        dispatch({
+            type: UPDATE_BREAKOUT_ROOMS,
+            rooms: {}
```

Comment:
reset the counter too

Reply:
maybe we can create an internal action _RESET_BREAKOUT_ROOMS` and apply the default state in the reducer. This way we make it future-proof, should we add more state in the future.